index.tsx

```diff
@@ -732,6 +737,36 @@ function anyScopeRegex(regex: RegExp) {
   }
 }
 
+function matchParagraph(source: string, state: MarkdownToJSX.State, prevCapturedString?: string) {
+  if (state._inline || state._simple) {
+    return null
+  }
+
+  if (prevCapturedString && !prevCapturedString.endsWith('\n')) { // don't match continuation of a line
+    return null
+  }
+
+  let match = ''
+
+  for (const line of source.split('\n')) {
```

Comment:
Can you use a different syntax that won't get transpiled into larger code? Maybe just a basic `for` loop

Reply:
Ack. Will do. (Just saw the autogenerated code, omg.)

Reply:
Maybe using `Array.every` instead would be best of both worlds?

Reply:
I'd rather just a standard `for` loop, it's not much longer and is usually an order faster.

Reply:
Nvm!